<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowsingSettingsPage</class>
 <widget class="QWidget" name="BrowsingSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browsing</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../data/data.qrc">
    <normaloff>:/icons/normal/globe.png</normaloff>:/icons/normal/globe.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="show_self">
       <property name="text">
        <string>Show self posts</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of pages to preload</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="preload_next">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>25</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Number of pages to keep in memory</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="max_preloaded_pages">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Network cache size</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="cache_size">
       <property name="suffix">
        <string> MB</string>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Save loved images to</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="loved_folder"/>
       </item>
       <item>
        <widget class="QPushButton" name="loved_browse">
         <property name="text">
          <string>Browse...</string>
         </property>
         <property name="icon">
          <iconset resource="../data/data.qrc">
           <normaloff>:/icons/normal/search.png</normaloff>:/icons/normal/search.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>249</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../data/data.qrc"/>
 </resources>
 <connections/>
</ui>
